src/DataStructures/Tensor/EagerMath/Magnitude.hpp

```diff
@@ -141,4 +170,17 @@ struct Sqrt : db::ComputeTag {
       sqrt_magnitude;
   using argument_tags = tmpl::list<Tag>;
 };
+
+/// \ingroup DataBoxTagsGroup
+/// \ingroup DataStructuresGroup
+/// The L2 norm of a tensor
+///
+/// \snippet Test_Magnitude.cpp sqrt_name
+template <typename Tag, bool Accumulate = false>
+struct L2Norm : db::ComputeTag {
```

Comment:
Perhaps we should have a compute item or at least a free function for the square of the L2 norm so that we can use it in `ObserveErrorNorms`. @wthrowe what do you think?

Reply:
Do we really want a free function to perform a `square` operation - isn't calling `square(db::get<L2Norm<Tag>>(box))` equivalent?

Reply:
An alternate could be to redefine these compute tags as `L2NormSq` and `PointwiseL2NormSq`, which can then be clubbed with the compute tag `Sqrt` to do what i want, or not to use it in `ObserveErrornorms`. thoughts?

Reply:
Just added a function for the square of the point-wise L2 norm.